tests/cli_doc/test_cli_doc_coverage.py

```
@@ -79,12 +79,12 @@ def test_documented_options_have_meta(self) -> None:  # noqa: PLR6301
                 + "\n\nAdd entries to CLI_OPTION_META in cli_options.py."
             )
 
-    def test_documented_options_not_excluded(self) -> None:  # noqa: PLR6301
-        """Verify that DOCUMENTED_OPTIONS are not in EXCLUDED_FROM_DOCS."""
-        overlap = DOCUMENTED_OPTIONS & EXCLUDED_FROM_DOCS
+    def test_documented_options_not_manual(self) -> None:  # noqa: PLR6301
```

Ruff (0.14.8)

Unused `noqa` directive (non-enabled: `PLR6301`)

Remove unused `noqa` directive

(RUF100)